vmsg.js

```diff
@@ -153,24 +298,17 @@ class Form {
     this.timer = null;
     this.audio = null;
     this.saveBtn = null;
-    this.audioCtx = null;
-    this.gainNode = null;
-    this.pitchFX = null;
-    this.encNode = null;
-    this.worker = null;
-    this.workerURL = null;
-    this.blob = null;
-    this.blobURL = null;
     this.tid = 0;
     this.start = 0;
     Object.seal(this);
 
-    this.initAudio()
+    this.recorder.initAudio()
+      .then(() => this.recorder.initWorker())
```

Comment:
Should be after `drawInit`

Reply:
Ahhhh I see -- fixed :)

Reply:
Sorry for insistence :) It's initAudio, then drawInit, then initWorker.
See https://github.com/Kagami/vmsg/blob/v0.2.0/vmsg.js#L155-L157